src/notifications/background/index.js

```diff
@@ -61,13 +64,21 @@ export default class NotificationBackground {
         return this.storage.fetchNotifById(id)
     }
 
+    async dispatchNotification(notifId) {
+        for (const notification of notifications.NOTIFS) {
```

Comment:
I would prefer you use array prototype methods here rather than loops for readability and consistency with the rest of the codebase. For example, this could be replaced with 

```ts
const notif = notifications.NOTIFS.find(notif => notif.id === notifId)
return this.storage.dispatchNotification(notif)
```

Although it's really not a big priority :)